utils/githooks/README.md

```diff
@@ -0,0 +1,19 @@
+# Git hooks
+
+This directory contains githooks that can be employed locally to
+automate things like copyright updates on modified files
+
+To use the pre-commit hook here, do the following locally
+
+```bash
+
+cat << EOF 2>&1 > .git/hooks/pre-commit
+#!/bin/sh
+
+# Invoke the githook for copyright validation
+exec utils/githooks/pre-commit 1>&2
+EOF
+
+chmod 700 .git/hooks/pre-commit
+
+```
```

Comment:
(lint) Quote this to prevent word splitting. [SC2046]
(lint) Use $(..) instead of legacy `..`. [SC2006]